src/util/serialise/index.js

```diff
@@ -28,29 +28,28 @@ export async function serialise (model, obj = {}, method = 'POST') {
 
     // Attributes and relationships
     for (let prop in obj) {
-      // Check if its a relationship
+      // Check if it's a relationship
       if (
         obj[prop] !== null &&
-        obj[prop].constructor === Object && (
-          typeof obj[prop].id === 'string' ||
-          typeof obj[prop].type === 'string'
-        )
+        obj[prop].constructor === Object
```

Comment:
I don't see how to merge those conditions and still maintain correct control flow, since it would run into the `else if (prop !== 'id')` branch and assume it's an attribute.

Reply:
Sorry I mean't

```js
if  (!== null && Object)
```

instead of the current

```js
if (
  !== null &&
  Object
)
```